docs/spelling_wordlist.txt

```diff
@@ -182,7 +182,7 @@ middleware
 middlewares
 miltidict
 misbehaviors
-misformed
+malformed
```

Comment:
The changed words here are presumably common dictionary words, so they can just be removed from the list. This is just a whitelist of additional words to add to the built-in dictionary when Sphinx builds the docs.

Reply:
ok, removing those